actors/miner/src/policy.rs

```diff
@@ -158,6 +162,10 @@ pub fn qa_power_for_sector(size: SectorSize, sector: &SectorOnChainInfo) -> Stor
     qa_power_for_weight(size, duration, &sector.deal_weight, &sector.verified_deal_weight)
 }
 
+pub fn base_power_for_sector(size: SectorSize) -> StoragePower {
+    BigInt::from(size as u64) * &*QUALITY_BASE_MULTIPLIER
```

Comment:
I don't think this multiplication is valid. My interpretation of QUALITY_BASE_MULTIPLIER is that it's supposed to be a _denominator_, when multiplying by either `DEAL_WEIGHT_MULTIPLIER` or `VERIFIED_DEAL_WEIGHT_MULTIPLIER`. In any case, multiplying size by 10 isn't what we want here.

Just do StoragePower::from(size) at the call sites.

Reply:
You are right. My math is invalid. I'd still like to keep the function `base_power_for_sector` so we have an clear way for getting CC sector power consistent with QA power function.